cam.go

```diff
@@ -288,7 +288,7 @@ func (fc *filteredCamera) images(ctx context.Context, extra map[string]interface
 	}
 
 	if !IsFromDataMgmt(ctx, extra) {
-		return images, meta, nil
+		return fc.lastCached(), meta, nil
```

Comment:
what about the metadata associated with the last image? This returns the metadata associated with the current image